doc/dev-manual-cn.md

```diff
@@ -99,3 +99,4 @@ Makefile rules:
 2. 然后通过 cpanm 来安装 test-gninx：`sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)`
 3. 然后 clone 最新的源码：`git clone https://github.com/openresty/test-nginx.git`
 4. 通过 perl 的 `prove` 命令来加载 test-nginx 的库，并运行 `/t` 目录下的测试案例集:`prove -Itest-nginx/lib -r t`
+>可通过 TEST_NGINX_BINARY 环境变量来设置执行的 nginx 路径,如 TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/67760965-4bbc5300-fa7d-11e9-9298-4c0da0629f73.png)

The comma character should be in tradition Chinese version.

Reply:
Also fix the semicolon character. https://github.com/apache/incubator-apisix/pull/755/commits/8550ed24d6992bac84524fd9353319878d242a53